<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="redo_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Redo</string>
    </property>
    <property name="text">
     <string>‚è©</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undo_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Undo</string>
    </property>
    <property name="text">
     <string>‚è™</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="toolTip">
     <string>About programm</string>
    </property>
    <property name="text">
     <string>‚ùî</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Open</string>
    </property>
    <property name="text">
     <string>üìÇ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Save</string>
    </property>
    <property name="text">
     <string>üíæ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="author_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="read_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Read Mode</string>
    </property>
    <property name="text">
     <string>üìñ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Create new</string>
    </property>
    <property name="text">
     <string>‚ûï</string>
    </property>
   </widget>
   <widget class="QPushButton" name="locale_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>switch lang</string>
    </property>
    <property name="text">
     <string>üá¨üáß</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>settings</string>
    </property>
    <property name="text">
     <string>‚öôÔ∏è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="style_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>change pallete</string>
    </property>
    <property name="text">
     <string>üåö</string>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>851</width>
      <height>551</height>
     </rect>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::TabbedView</enum>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="tabsMovable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
